### ISSUE TITLE
Swapped pins in ABK-PRO-MINI

### ISSUE THREAD

**Issue by nitram147** [cyberlink1/Arduino-Eagle-Cad-Library#3]
Hello, in ABK-PRO-MINI you have swapped pins it should be (from top left to bottom left) - GND, GND, VCC, RX1, TX0, DTR but you have DTR, TX0, RX1, VCC, GND, GND.

**Comment by nitram147** [comment]
And between pin A7 and D10 is bigger gap than between A7 and A6. But in reality on real arduino clone board that gap between A7 and D10 have the sime size as the gap between A7 and A6.

**Comment by jk987** [comment]
Also pins A0-A3 seem to be swapped.
More precisely, in schematics they are correct but on the board they are swapped.

**Comment by nitram147** [comment]
You are right. Boards from china arrived (boards in which I had used this library) and pins are swapped so I had to rewrite my code ...

**Comment by chupocro** [comment]
Top pins are mirrored. A0 - A3 pins are mirrored on board. 'RX1' and 'TX0' should be RXI and TXO as in RX **I**nput, TX **O**utput. GND pins are not in sync with the schematics too, for example .sch GND4 pin is at the top and .brd GND4 pin is at the bottom. Although I don't have ABK version of the board I can see in the online pictures the gap between A7 and A10 really **is** bigger on that board. There is other similar board (marked 'The Simple') where the gap is not bigger. Since the library should be edited anyway (because of the swapped pins) then at the same time the gap between A7 and 10 could be adjusted as needed.

EDIT:
GND pins should be confirured as **power** pins (instead as I/O pins)  and the names should be: GND@1, GND@2, ... in order to connections could be routed correctly. The same naming convention should be used for multiple RX, TX, RST and Vcc pins.
